### PR TITLE
Add e2e tests for brush filters in line chart

### PR DESCRIPTION
**Pull request by uladzimirdev**
Relates to https://github.com/metabase/metabase/issues/37104

### Description

Add e2e tests for line chart filters:

- using dragging to select some area
- using drill to select a single dot

Stress test run https://github.com/metabase/metabase/actions/runs/7356917227

### How to verify

tests should pass

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js [hunk 1]
```diff
@@ -577,6 +577,45 @@ describe("scenarios > visualizations > line chart", () => {
       cy.get(".y-axis-label").invoke("text").should("eq", "Average of Price");
     });
   });
+
+  describe("brush filters", () => {
+    it("should apply filters to the series selecting area range", () => {
+      cy.viewport(1280, 800);
```
Comment (uladzimirdev) [inline e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js:583]: without viewport result is not determenistic

File: e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js [hunk 2]
```diff
@@ -577,6 +577,45 @@ describe("scenarios > visualizations > line chart", () => {
...
+
+      cy.findByTestId("filter-pill").should(
+        "have.text",
+        "Created At is Apr 1, 12:00 AM – Sep 1, 2022, 12:00 AM",
+      );
+
+      cy.get(".Visualization .dot").should("have.length", 6);
+    });
+
+    it("should apply filter to the series after drilling down", () => {
```
Comment (ranquild) [inline e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js:605]: Technically this is not a brush filter - it's a regular drill

**Review by ranquild — APPROVED** [review]

File: e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js (hunk 2 above)
Comment (uladzimirdev) [reply e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js:605]: should I keep it or it's better to remove it? I can add `(drill)` in the name

Comment (uladzimirdev) [reply e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js]: removed this test, I'll add another test where x-axis is number in a different PR

**Pull request merged**